knit/src/Knit/Autocompletion.hs

```diff
@@ -130,17 +55,16 @@ suggestionExprs procs = goExpr
       lit@(ExprLit _ _) -> pure lit
       ExprProcCall NoExt p -> ExprProcCall NoExt <$> goPc p
       XExpr (ExprInBrackets l e r) -> do
-        oldLeft <- liftLeft get
-        oldRight <- liftRight get
-        liftLeft $ put $ LeftSpace $ fbSpace l
-        liftRight $ put $ RightSpace $ fbSpace r
+        oldLeft <- use leftSpace
```

Comment:
Is there reason to have `oldLeft` and `oldRight` separately as opposed to `oldSuggestionCtx <- get`?

Reply:
missed it